src/helpers/errors.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ *  SudoSOS back-end API service.
+ *  Copyright (C) 2024  Study association GEWIS
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as published
+ *  by the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+export class NotImplementedError extends Error {
+  constructor(message?: string) {
+    super(message);
+    this.name = 'NotImplementedError';
+
+    // Ensure the error stack trace includes this constructor call
+    if (Error.captureStackTrace) {
+      Error.captureStackTrace(this, NotImplementedError);
+    }
+  }
+}
```

Comment:
Maybe make this a seperate file called `NotImplementedError` in the folder `src/helpers/errors/`, because then you can create an `index.ts` that exports `NotImplementedError`. You achieve the same results, but can easily organize new custom errors we might create in the future.